Aboutme.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Me</title>
+    <link rel="stylesheet" href="style2.css" />
+    <script src="https://kit.fontawesome.com/ca44700445.js" crossorigin="anonymous"></script>
+</head>
+<body>
+
+    <h1>About Me</h1>
+
+    <p> 
+        I’m a 22 year old guy from a town in Wantage, who moved to Coventry to study Mathematics at the University of Warwick. Having spent 3 years there hurting my brain, I thought I would give my brain a little rest and take a gap year volunteering for my Church and helping with the homeless. Well lockdown hit and so when the year ended I was eager to get back into hurting my brain. Hello School of Code!
```

Comment:
Breaking up sentences onto new lines is easily achievable with a <br /> tag. Paragraphs can be hard to read if they are stretched across browser on one line. This is subjective so just for your consideration.